public/app/features/dashboard/components/Inspector/InspectHeader.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC } from 'react';
+import { css } from 'emotion';
+import { Icon, stylesFactory, Tab, TabsBar, useTheme } from '@grafana/ui';
+import { GrafanaTheme, SelectableValue } from '@grafana/data';
+import { InspectTab } from './PanelInspector';
+import { PanelModel } from '../../state';
+
+interface Props {
+  tab: InspectTab;
+  tabs: Array<{ label: string; value: InspectTab }>;
+  stats: { requestTime: number; queries: number; dataSources: number };
+  panel: PanelModel;
+  isExpanded: boolean;
+
+  onSelectTab: (tab: SelectableValue<InspectTab>) => void;
+  onClose: () => void;
+  onToggleExpand: () => void;
+}
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  return {
+    header: css`
+      background-color: ${theme.colors.formInputBg};
+      z-index: 1;
+      flex-grow: 0;
+      padding: ${theme.spacing.sm} ${theme.spacing.sm} 0 ${theme.spacing.lg};
+    `,
+    actions: css`
+      display: flex;
+      align-items: baseline;
+      justify-content: space-between;
+      margin-bottom: ${theme.spacing.md};
+      font-size: ${theme.typography.size.lg};
+    `,
+    expand: css`
+      cursor: pointer;
+    `,
+    close: css`
+      cursor: pointer;
+      width: 25px;
+      height: 100%;
+      display: flex;
+      flex-shrink: 0;
+      justify-content: center;
+    `,
+    titleWrapper: css`
+      margin-bottom: ${theme.spacing.lg};
+    `,
+  };
+});
+
+export const InspectHeader: FC<Props> = ({
+  tab,
+  tabs,
+  onSelectTab,
+  onClose,
+  onToggleExpand,
+  panel,
+  stats,
+  isExpanded,
+}) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+  const statsString = `${stats.queries} ${stats.queries === 1 ? 'query' : 'queries'} - ${stats.dataSources} ${
```

Comment:
nit: Would it be a good idea to create a private function to handle the creation of this string? I'm thinking so we can prevent multiple ternary operation in the same assignment and make it a bit more readable.